README.md

```diff
@@ -1,3 +1,5 @@
 # PR Checklist
 
 This checklist is a set of guidelines for every pull request you make. Not every check will be perfectly relevant, but it should give you a bit more confidence that your code and PR are truly ready for the eyes of other developers.
+
+This change is literally just to take a screenshot of a comment.
```

Comment:
I chose to wrap the multiple database ops here in a single transaction. I’m not sure if this is the most efficient way to process this. It ensures all-or-nothing inserts for this set of related records, but it might take longer. Any suggestions for alternatives here?